### PR TITLE
feat: update empathy dependencies

### PR DESCRIPTION
**Pull request by javieri-empathy**
EX-6279

Updates Empathy dependencies to its last version.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -5,12 +5,12 @@
     <DeviceDetector @DeviceProvided="$setLocaleDevice" :breakpoints="breakpoints" />
     <Tagging />
     <UrlHandler env="env" />
-    <XModal v-if="isOpen" />
+    <XModal v-if="isOpen" :openEvents="openEvents" />
```
Comment (javieri-empathy) [inline src/App.vue:8]: The only issue I found is this one. I think this XModal should be moved to x-components. And modals, like panels should have its own module.

Comment (LuisMartinez15) [reply src/App.vue:8]: I think there is already a task to create an `XModal` in `x-components` that is going to serve as the main container of the application.

**Review by LuisMartinez15 — DISMISSED** [review]
good job!

**Review by LuisMartinez15 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -24,11 +24,11 @@
     "csv:json": "csv-json ./output ./src/i18n/messages"
   },
   "dependencies": {
-    "@empathyco/x-adapter": "^7.0.0-alpha.27",
-    "@empathyco/x-components": "^3.0.0-alpha.91",
-    "@empathyco/x-deep-merge": "^1.3.0-alpha.12",
-    "@empathyco/x-types": "^10.0.0-alpha.21",
-    "@empathyco/x-utils": "^0.1.0-alpha.11",
+    "@empathyco/x-adapter": "^7.0.0-alpha.33",
+    "@empathyco/x-components": "^3.0.0-alpha.111",
```
Comment (tiborux) [inline package.json:28]:
```suggestion
    "@empathyco/x-components": "^3.0.0-alpha.112",
```

**Review by tiborux — APPROVED** [review]

**Pull request merged**